### PR TITLE
Add future dates

### LINKED ISSUE #42: Dev CoP - All Presentation/Workshop Dates
Session | Pres/Workshop | Date | Time | Topic  
-- | -- | -- | -- | --
Session   1 | Presentation | Tuesday,   September 10th | 10am | #1 Move Source Control to Git
  | Workshop | Tuesday,   September 17th | 12pm | #1 Move Source Control to Git
Session   2 | Presentation | Tuesday,   October 15th | 10am | #10 Make use of the GitFlow branching model
  | Workshop | Monday   October 21st | 12pm | #10 Make use of the GitFlow branching model
Session   3 | Presentation | Tuesday   November 12th | 10am | TBD
  | Workshop | Tuesday   November 19th | 12pm | TBD
Session   4 | Presentation | Tuesday   December 3rd  | 10am | TBD
  | Workshop | Tuesday   December 10th | 12pm | TBD
Session   5 | Presentation | Tuesday   January 14th | 10am | TBD
  | Workshop | Tuesday   January 21st | 12pm | TBD
Session   6 | Presentation | Tuesday   February 4th | 10am | TBD
  | Workshop | Tuesday   February 11th | 12pm | TBD
Session   7 | Presentation | Tuesday   March 3rd | 10am | TBD
  | Workshop | Tuesday   March 10th | 12pm | TBD
Session   8 | Presentation | Tuesday   April 7th | 10am | TBD
  | Workshop | Tuesday   April 14th | 12pm | TBD
Session   9 | Presentation | Tuesday   May 5th | 10am | TBD
  | Workshop | Tuesday   May 12th | 12pm | TBD
Session   10 | Presentation | Tuesday   June 2nd | 10am | TBD
  | Workshop | Tuesday   June 9th | 12pm | TBD

### PR DESCRIPTION
**Pull request by ahmad-shahid**
Added table with future presentation/workshop dates

Closes #42

### CODE REVIEW

File: strategy.md [hunk 1]
```diff
@@ -35,3 +35,28 @@ Working with management on what they perceive to be preventing them from accepti
 ## Community Building & Engagement
 
 We will also focus on keeping the community engaged and involved in the various activities through social media and advertising methods.
+
+## Session Dates
```
Comment (s-laugh) [inline strategy.md]: I think it might be useful to track this on a separate page.

File: strategy.md [hunk 2]
```diff
@@ -35,3 +35,28 @@ Working with management on what they perceive to be preventing them from accepti
...
+
+Session | Pres/Workshop | Date | Time | Topic  
+-- | -- | -- | -- | --
+Session   1 | Presentation | Tuesday,   September 10th | 10am | Move Source Control to Git
+  -| Workshop | Tuesday,   September 17th | 12pm | Move Source Control to Git
+Session   2 | Presentation | Tuesday,   October 15th | 10am | Make use of the GitFlow branching model
+  -| Workshop | Monday   October 21st | 12pm | Make use of the GitFlow branching model
+Session   3 | Presentation | Tuesday   November 12th | 10am | TBD
```
Comment (s-laugh) [inline strategy.md]:
```suggestion
Session   3 | Presentation | Tuesday   November 12th | 10am | Establish code review #3
```

File: strategy.md [hunk 3]
```diff
@@ -35,3 +35,28 @@ Working with management on what they perceive to be preventing them from accepti
...
+  -| Workshop | Tuesday   November 19th | 12pm | TBD
```
Comment (s-laugh) [inline strategy.md]:
```suggestion
  -| Workshop | Tuesday   November 19th | 12pm | Establish code review #3
```

File: strategy.md (hunk 1 above)
Comment (ahmad-shahid) [reply strategy.md]: Created a separate page for the dates and added a link from the strategy page.

File: dates.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Here are the dates for the Presentations and Workshops for 2019/20. 
+
+Session | Pres/Workshop | Date | Time | Topic  
+-- | -- | -- | -- | --
+Session   1 | Presentation | Tuesday,   September 10th | 10am | Move Source Control to Git
+  -| Workshop | Tuesday,   September 17th | 12pm | Move Source Control to Git
+Session   2 | Presentation | Tuesday,   October 15th | 10am | Make use of the GitFlow branching model
+  -| Workshop | Monday   October 21st | 12pm | Make use of the GitFlow branching model
+Session   3 | Presentation | Tuesday   November 12th | 10am | TBD
```
Comment (s-laugh) [inline dates.md]:
```suggestion
Session   3 | Presentation | Tuesday   November 12th | 10am | Establish code review #3
```

File: dates.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  -| Workshop | Tuesday   November 19th | 12pm | TBD
```
Comment (s-laugh) [inline dates.md]:
```suggestion
  -| Workshop | Tuesday   November 19th | 12pm | Establish code review #3
```

**Comment by s-laugh** [comment]
Looks like you have that signing issue again with commits. 

Follow this to re-sign the commits. Let me know if you need help.
https://superuser.com/questions/397149/can-you-gpg-sign-old-commits

**Pull request merged**